device-types/Mellanox/SX6036.yml

```diff
@@ -8,10 +8,10 @@ console-ports:
   - name: Console
     type: rj-45
 power-ports:
-  - name: psu1
+  - name: PS1
     type: iec-60320-c14
     maximum_draw: 500
-  - name: psu2
+  - name: PS2
     type: iec-60320-c14
     maximum_draw: 500
```

Comment:
Please correct me if I am wrong, but it looks like the PSUs are modular. If that is true, these should be defined as `module-bays` and the associated PSUs should be created in the module-types folder